packages/xpetra/cmake/Xpetra_config.hpp.in

```diff
@@ -24,8 +24,6 @@
 
 #cmakedefine HAVE_XPETRA_KOKKOSCONTAINERS
 
-#cmakedefine HAVE_XPETRA_KOKKOSCORE
```

Comment:
We might want to define `#cmakedefine HAVE_XPETRA_KOKKOSKERNELS`?

Reply:
addressed